test/small.jl

```diff
@@ -181,18 +181,20 @@ end;
             ),
         )
 
-        test_bicoloring_decompression(
-            A,
-            problem,
-            GreedyColoringAlgorithm{:direct}(order; postprocessing=true);
-            test_fast=true,
-        )
+        @testset "postprocessing_minimizes = $target" for target in (:all_colors, :row_colors, :column_colors)
```

Comment:
This should be added to the random tests too

Reply:
Done in this commit: https://github.com/gdalle/SparseMatrixColorings.jl/pull/267/commits/5f40c3dc5eb60a69bef0e3698ac49df1d1c2ab1f